automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -44,6 +44,7 @@ gcloud:
   device:
    - model: Pixel2
      version: 28
+     locale: en_US
```

Comment:
With `en` as the Firebase default, I wonder if some generic not `en_US`. I guess that's fine to add here, but I don't know.

```Log
% gcloud firebase test android locales list 
┌────────┬─────────────────────────────┬───────────────────────────────────┬─────────┐
│ LOCALE │             NAME            │               REGION              │   TAGS  │
├────────┼─────────────────────────────┼───────────────────────────────────┼─────────┤
│ af     │ Afrikaans                   │                                   │         │
│ af_NA  │ Afrikaans                   │ Namibia                           │         │
│ af_ZA  │ Afrikaans                   │ South Africa                      │         │
│ agq    │ Aghem                       │                                   │         │
│ agq_CM │ Aghem                       │ Cameroon                          │         │
│ ak     │ Akan                        │                                   │         │
│ ak_GH  │ Akan                        │ Ghana                             │         │
│ am     │ Amharic                     │                                   │         │
│ am_ET  │ Amharic                     │ Ethiopia                          │         │
│ ar     │ Arabic                      │                                   │         │
│ ar_001 │ Arabic                      │ World                             │         │
│ ar_AE  │ Arabic                      │ United Arab Emirates              │         │
│ ar_BH  │ Arabic                      │ Bahrain                           │         │
│ ar_DJ  │ Arabic                      │ Djibouti                          │         │
│ ar_DZ  │ Arabic                      │ Algeria                           │         │
│ ar_EG  │ Arabic                      │ Egypt                             │         │
│ ar_EH  │ Arabic                      │ Western Sahara                    │         │
│ ar_ER  │ Arabic                      │ Eritrea                           │         │
│ ar_IL  │ Arabic                      │ Israel                            │         │
│ ar_IQ  │ Arabic                      │ Iraq                              │         │
│ ar_JO  │ Arabic                      │ Jordan                            │         │
│ ar_KM  │ Arabic                      │ Comoros                           │         │
│ ar_KW  │ Arabic                      │ Kuwait                            │         │
│ ar_LB  │ Arabic                      │ Lebanon                           │         │
│ ar_LY  │ Arabic                      │ Libya                             │         │
│ ar_MA  │ Arabic                      │ Morocco                           │         │
│ ar_MR  │ Arabic                      │ Mauritania                        │         │
│ ar_OM  │ Arabic                      │ Oman                              │         │
│ ar_PS  │ Arabic                      │ Palestine                         │         │
│ ar_QA  │ Arabic                      │ Qatar                             │         │
│ ar_SA  │ Arabic                      │ Saudi Arabia                      │         │
│ ar_SD  │ Arabic                      │ Sudan                             │         │
│ ar_SO  │ Arabic                      │ Somalia                           │         │
│ ar_SS  │ Arabic                      │ South Sudan                       │         │
│ ar_SY  │ Arabic                      │ Syria                             │         │
│ ar_TD  │ Arabic                      │ Chad                              │         │
│ ar_TN  │ Arabic                      │ Tunisia                           │         │
│ ar_YE  │ Arabic                      │ Yemen                             │         │
│ as     │ Assamese                    │                                   │         │
│ as_IN  │ Assamese                    │ India                             │         │
│ asa    │ Asu                         │                                   │         │
│ asa_TZ │ Asu                         │ Tanzania                          │         │
│ az     │ Azerbaijani                 │                                   │         │
│ az_AZ  │ Azerbaijani                 │ Azerbaijan                        │         │
│ bas    │ Basaa                       │                                   │         │
│ bas_CM │ Basaa                       │ Cameroon                          │         │
│ be     │ Belarusian                  │                                   │         │
│ be_BY  │ Belarusian                  │ Belarus                           │         │
│ bem    │ Bemba                       │                                   │         │
│ bem_ZM │ Bemba                       │ Zambia                            │         │
│ bez    │ Bena                        │                                   │         │
│ bez_TZ │ Bena                        │ Tanzania                          │         │
│ bg     │ Bulgarian                   │                                   │         │
│ bg_BG  │ Bulgarian                   │ Bulgaria                          │         │
│ bm     │ Bambara                     │                                   │         │
│ bm_ML  │ Bambara                     │ Mali                              │         │
│ bn     │ Bengali                     │                                   │         │
│ bn_BD  │ Bengali                     │ Bangladesh                        │         │
│ bn_IN  │ Bengali                     │ India                             │         │
│ bo     │ Tibetan                     │                                   │         │
│ bo_CN  │ Tibetan                     │ China                             │         │
│ bo_IN  │ Tibetan                     │ India                             │         │
│ br     │ Breton                      │                                   │         │
│ br_FR  │ Breton                      │ France                            │         │
│ brx    │ Bodo                        │                                   │         │
│ brx_IN │ Bodo                        │ India                             │         │
│ bs     │ Bosnian                     │                                   │         │
│ bs_BA  │ Bosnian                     │ Bosnia and Herzegovina            │         │
│ ca     │ Catalan                     │                                   │         │
│ ca_AD  │ Catalan                     │ Andorra                           │         │
│ ca_ES  │ Catalan                     │ Spain                             │         │
│ ca_FR  │ Catalan                     │ France                            │         │
│ ca_IT  │ Catalan                     │ Italy                             │         │
│ cgg    │ Chiga                       │                                   │         │
│ cgg_UG │ Chiga                       │ Uganda                            │         │
│ chr    │ Cherokee                    │                                   │         │
│ chr_US │ Cherokee                    │ United States                     │         │
│ cs     │ Czech                       │                                   │         │
│ cs_CZ  │ Czech                       │ Czech Republic                    │         │
│ cy     │ Welsh                       │                                   │         │
│ cy_GB  │ Welsh                       │ United Kingdom                    │         │
│ da     │ Danish                      │                                   │         │
│ da_DK  │ Danish                      │ Denmark                           │         │
│ da_GL  │ Danish                      │ Greenland                         │         │
│ dav    │ Taita                       │                                   │         │
│ dav_KE │ Taita                       │ Kenya                             │         │
│ de     │ German                      │                                   │         │
│ de_AT  │ German                      │ Austria                           │         │
│ de_BE  │ German                      │ Belgium                           │         │
│ de_CH  │ German                      │ Switzerland                       │         │
│ de_DE  │ German                      │ Germany                           │         │
│ de_LI  │ German                      │ Liechtenstein                     │         │
│ de_LU  │ German                      │ Luxembourg                        │         │
│ dje    │ Zarma                       │                                   │         │
│ dje_NE │ Zarma                       │ Niger                             │         │
│ dua    │ Duala                       │                                   │         │
│ dua_CM │ Duala                       │ Cameroon                          │         │
│ dyo    │ Jola-Fonyi                  │                                   │         │
│ dyo_SN │ Jola-Fonyi                  │ Senegal                           │         │
│ dz     │ Dzongkha                    │                                   │         │
│ dz_BT  │ Dzongkha                    │ Bhutan                            │         │
│ ebu    │ Embu                        │                                   │         │
│ ebu_KE │ Embu                        │ Kenya                             │         │
│ ee     │ Ewe                         │                                   │         │
│ ee_GH  │ Ewe                         │ Ghana                             │         │
│ ee_TG  │ Ewe                         │ Togo                              │         │
│ el     │ Greek                       │                                   │         │
│ el_CY  │ Greek                       │ Cyprus                            │         │
│ el_GR  │ Greek                       │ Greece                            │         │
│ en     │ English                     │                                   │ default │
│ en_001 │ English                     │ World                             │         │
│ en_150 │ English                     │ Europe                            │         │
│ en_AG  │ English                     │ Antigua and Barbuda               │         │
│ en_AI  │ English                     │ Anguilla                          │         │
│ en_AS  │ English                     │ American Samoa                    │         │
│ en_AU  │ English                     │ Australia                         │         │
│ en_BB  │ English                     │ Barbados                          │         │
│ en_BE  │ English                     │ Belgium                           │         │
│ en_BM  │ English                     │ Bermuda                           │         │
│ en_BS  │ English                     │ Bahamas                           │         │
│ en_BW  │ English                     │ Botswana                          │         │
│ en_BZ  │ English                     │ Belize                            │         │
│ en_CA  │ English                     │ Canada                            │         │
│ en_CC  │ English                     │ Cocos (Keeling) Islands           │         │
│ en_CK  │ English                     │ Cook Islands                      │         │
│ en_CM  │ English                     │ Cameroon                          │         │
│ en_CX  │ English                     │ Christmas Island                  │         │
│ en_DG  │ English                     │ Diego Garcia                      │         │
│ en_DM  │ English                     │ Dominica                          │         │
│ en_ER  │ English                     │ Eritrea                           │         │
│ en_FJ  │ English                     │ Fiji                              │         │
│ en_FK  │ English                     │ Falkland Islands (Islas Malvinas) │         │
│ en_FM  │ English                     │ Micronesia                        │         │
│ en_GB  │ English                     │ United Kingdom                    │         │
│ en_GD  │ English                     │ Grenada                           │         │
│ en_GG  │ English                     │ Guernsey                          │         │
│ en_GH  │ English                     │ Ghana                             │         │
│ en_GI  │ English                     │ Gibraltar                         │         │
│ en_GM  │ English                     │ Gambia                            │         │
│ en_GU  │ English                     │ Guam                              │         │
│ en_GY  │ English                     │ Guyana                            │         │
│ en_HK  │ English                     │ Hong Kong                         │         │
│ en_IE  │ English                     │ Ireland                           │         │
│ en_IM  │ English                     │ Isle of Man                       │         │
│ en_IN  │ English                     │ India                             │         │
│ en_IO  │ English                     │ British Indian Ocean Territory    │         │
│ en_JE  │ English                     │ Jersey                            │         │
│ en_JM  │ English                     │ Jamaica                           │         │
│ en_KE  │ English                     │ Kenya                             │         │
│ en_KI  │ English                     │ Kiribati                          │         │
│ en_KN  │ English                     │ Saint Kitts and Nevis             │         │
│ en_KY  │ English                     │ Cayman Islands                    │         │
│ en_LC  │ English                     │ Saint Lucia                       │         │
│ en_LR  │ English                     │ Liberia                           │         │
│ en_LS  │ English                     │ Lesotho                           │         │
│ en_MG  │ English                     │ Madagascar                        │         │
│ en_MH  │ English                     │ Marshall Islands                  │         │
│ en_MO  │ English                     │ Macau                             │         │
│ en_MP  │ English                     │ Northern Mariana Islands          │         │
│ en_MS  │ English                     │ Montserrat                        │         │
│ en_MT  │ English                     │ Malta                             │         │
│ en_MU  │ English                     │ Mauritius                         │         │
│ en_MW  │ English                     │ Malawi                            │         │
│ en_NA  │ English                     │ Namibia                           │         │
│ en_NF  │ English                     │ Norfolk Island                    │         │
│ en_NG  │ English                     │ Nigeria                           │         │
│ en_NR  │ English                     │ Nauru                             │         │
│ en_NU  │ English                     │ Niue                              │         │
│ en_NZ  │ English                     │ New Zealand                       │         │
│ en_PG  │ English                     │ Papua New Guinea                  │         │
│ en_PH  │ English                     │ Philippines                       │         │
│ en_PK  │ English                     │ Pakistan                          │         │
│ en_PN  │ English                     │ Pitcairn Islands                  │         │
│ en_PR  │ English                     │ Puerto Rico                       │         │
│ en_PW  │ English                     │ Palau                             │         │
│ en_RW  │ English                     │ Rwanda                            │         │
│ en_SB  │ English                     │ Solomon Islands                   │         │
│ en_SC  │ English                     │ Seychelles                        │         │
│ en_SD  │ English                     │ Sudan                             │         │
│ en_SG  │ English                     │ Singapore                         │         │
│ en_SH  │ English                     │ Saint Helena                      │         │
│ en_SL  │ English                     │ Sierra Leone                      │         │
│ en_SS  │ English                     │ South Sudan                       │         │
│ en_SX  │ English                     │ Sint Maarten                      │         │
│ en_SZ  │ English                     │ Swaziland                         │         │
│ en_TC  │ English                     │ Turks and Caicos Islands          │         │
│ en_TK  │ English                     │ Tokelau                           │         │
│ en_TO  │ English                     │ Tonga                             │         │
│ en_TT  │ English                     │ Trinidad and Tobago               │         │
│ en_TV  │ English                     │ Tuvalu                            │         │
│ en_TZ  │ English                     │ Tanzania                          │         │
│ en_UG  │ English                     │ Uganda                            │         │
│ en_UM  │ English                     │ U.S. Outlying Islands             │         │
│ en_US  │ English                     │ United States                     │         │
│ en_VC  │ English                     │ St. Vincent & Grenadines          │         │
│ en_VG  │ English                     │ British Virgin Islands            │         │
│ en_VI  │ English                     │ U.S. Virgin Islands               │         │
│ en_VU  │ English                     │ Vanuatu                           │         │
│ en_WS  │ English                     │ Samoa                             │         │
│ en_ZA  │ English                     │ South Africa                      │         │
│ en_ZM  │ English                     │ Zambia                            │         │
│ en_ZW  │ English                     │ Zimbabwe                          │         │
│ eo     │ Esperanto                   │                                   │         │
│ es     │ Spanish                     │                                   │         │
│ es_419 │ Spanish                     │ Latin America                     │         │
│ es_AR  │ Spanish                     │ Argentina                         │         │
│ es_BO  │ Spanish                     │ Bolivia                           │         │
│ es_CL  │ Spanish                     │ Chile                             │         │
│ es_CO  │ Spanish                     │ Colombia                          │         │
│ es_CR  │ Spanish                     │ Costa Rica                        │         │
│ es_CU  │ Spanish                     │ Cuba                              │         │
│ es_DO  │ Spanish                     │ Dominican Republic                │         │
│ es_EA  │ Spanish                     │ Ceuta and Melilla                 │         │
│ es_EC  │ Spanish                     │ Ecuador                           │         │
│ es_ES  │ Spanish                     │ Spain                             │         │
│ es_GQ  │ Spanish                     │ Equatorial Guinea                 │         │
│ es_GT  │ Spanish                     │ Guatemala                         │         │
│ es_HN  │ Spanish                     │ Honduras                          │         │
│ es_IC  │ Spanish                     │ Canary Islands                    │         │
│ es_MX  │ Spanish                     │ Mexico                            │         │
│ es_NI  │ Spanish                     │ Nicaragua                         │         │
│ es_PA  │ Spanish                     │ Panama                            │         │
│ es_PE  │ Spanish                     │ Peru                              │         │
│ es_PH  │ Spanish                     │ Philippines                       │         │
│ es_PR  │ Spanish                     │ Puerto Rico                       │         │
│ es_PY  │ Spanish                     │ Paraguay                          │         │
│ es_SV  │ Spanish                     │ El Salvador                       │         │
│ es_US  │ Spanish                     │ United States                     │         │
│ es_UY  │ Spanish                     │ Uruguay                           │         │
│ es_VE  │ Spanish                     │ Venezuela                         │         │
│ et     │ Estonian                    │                                   │         │
│ et_EE  │ Estonian                    │ Estonia                           │         │
│ eu     │ Basque                      │                                   │         │
│ eu_ES  │ Basque                      │ Spain                             │         │
│ ewo    │ Ewondo                      │                                   │         │
│ ewo_CM │ Ewondo                      │ Cameroon                          │         │
│ fa     │ Persian                     │                                   │         │
│ fa_AF  │ Persian                     │ Afghanistan                       │         │
│ fa_IR  │ Persian                     │ Iran                              │         │
│ ff     │ Fulah                       │                                   │         │
│ ff_SN  │ Fulah                       │ Senegal                           │         │
│ fi     │ Finnish                     │                                   │         │
│ fi_FI  │ Finnish                     │ Finland                           │         │
│ fil    │ Filipino                    │                                   │         │
│ fil_PH │ Filipino                    │ Philippines                       │         │
│ fo     │ Faroese                     │                                   │         │
│ fo_FO  │ Faroese                     │ Faroe Islands                     │         │
│ fr     │ French                      │                                   │         │
│ fr_BE  │ French                      │ Belgium                           │         │
│ fr_BF  │ French                      │ Burkina Faso                      │         │
│ fr_BI  │ French                      │ Burundi                           │         │
│ fr_BJ  │ French                      │ Benin                             │         │
│ fr_BL  │ French                      │ Saint Barthélemy                  │         │
│ fr_CA  │ French                      │ Canada                            │         │
│ fr_CD  │ French                      │ Congo (DRC)                       │         │
│ fr_CF  │ French                      │ Central African Republic          │         │
│ fr_CG  │ French                      │ Congo (Republic)                  │         │
│ fr_CH  │ French                      │ Switzerland                       │         │
│ fr_CI  │ French                      │ Côte d’Ivoire                     │         │
│ fr_CM  │ French                      │ Cameroon                          │         │
│ fr_DJ  │ French                      │ Djibouti                          │         │
│ fr_DZ  │ French                      │ Algeria                           │         │
│ fr_FR  │ French                      │ France                            │         │
│ fr_GA  │ French                      │ Gabon                             │         │
│ fr_GF  │ French                      │ French Guiana                     │         │
│ fr_GN  │ French                      │ Guinea                            │         │
│ fr_GP  │ French                      │ Guadeloupe                        │         │
│ fr_GQ  │ French                      │ Equatorial Guinea                 │         │
│ fr_HT  │ French                      │ Haiti                             │         │
│ fr_KM  │ French                      │ Comoros                           │         │
│ fr_LU  │ French                      │ Luxembourg                        │         │
│ fr_MA  │ French                      │ Morocco                           │         │
│ fr_MC  │ French                      │ Monaco                            │         │
│ fr_MF  │ French                      │ Saint Martin                      │         │
│ fr_MG  │ French                      │ Madagascar                        │         │
│ fr_ML  │ French                      │ Mali                              │         │
│ fr_MQ  │ French                      │ Martinique                        │         │
│ fr_MR  │ French                      │ Mauritania                        │         │
│ fr_MU  │ French                      │ Mauritius                         │         │
│ fr_NC  │ French                      │ New Caledonia                     │         │
│ fr_NE  │ French                      │ Niger                             │         │
│ fr_PF  │ French                      │ French Polynesia                  │         │
│ fr_PM  │ French                      │ Saint Pierre and Miquelon         │         │
│ fr_RE  │ French                      │ Réunion                           │         │
│ fr_RW  │ French                      │ Rwanda                            │         │
│ fr_SC  │ French                      │ Seychelles                        │         │
│ fr_SN  │ French                      │ Senegal                           │         │
│ fr_SY  │ French                      │ Syria                             │         │
│ fr_TD  │ French                      │ Chad                              │         │
│ fr_TG  │ French                      │ Togo                              │         │
│ fr_TN  │ French                      │ Tunisia                           │         │
│ fr_VU  │ French                      │ Vanuatu                           │         │
│ fr_WF  │ French                      │ Wallis and Futuna                 │         │
│ fr_YT  │ French                      │ Mayotte                           │         │
│ ga     │ Irish                       │                                   │         │
│ ga_IE  │ Irish                       │ Ireland                           │         │
│ gl     │ Galician                    │                                   │         │
│ gl_ES  │ Galician                    │ Spain                             │         │
│ gsw    │ Swiss German                │                                   │         │
│ gsw_CH │ Swiss German                │ Switzerland                       │         │
│ gsw_LI │ Swiss German                │ Liechtenstein                     │         │
│ gu     │ Gujarati                    │                                   │         │
│ gu_IN  │ Gujarati                    │ India                             │         │
│ guz    │ Gusii                       │                                   │         │
│ guz_KE │ Gusii                       │ Kenya                             │         │
│ gv     │ Manx                        │                                   │         │
│ gv_IM  │ Manx                        │ Isle of Man                       │         │
│ ha     │ Hausa                       │                                   │         │
│ ha_GH  │ Hausa                       │ Ghana                             │         │
│ ha_NE  │ Hausa                       │ Niger                             │         │
│ ha_NG  │ Hausa                       │ Nigeria                           │         │
│ haw    │ Hawaiian                    │                                   │         │
│ haw_US │ Hawaiian                    │ United States                     │         │
│ iw     │ Hebrew                      │                                   │         │
│ iw_IL  │ Hebrew                      │ Israel                            │         │
│ hi     │ Hindi                       │                                   │         │
│ hi_IN  │ Hindi                       │ India                             │         │
│ hr     │ Croatian                    │                                   │         │
│ hr_BA  │ Croatian                    │ Bosnia and Herzegovina            │         │
│ hr_HR  │ Croatian                    │ Croatia                           │         │
│ hu     │ Hungarian                   │                                   │         │
│ hu_HU  │ Hungarian                   │ Hungary                           │         │
│ hy     │ Armenian                    │                                   │         │
│ hy_AM  │ Armenian                    │ Armenia                           │         │
│ in     │ Indonesian                  │                                   │         │
│ in_ID  │ Indonesian                  │ Indonesia                         │         │
│ ig     │ Igbo                        │                                   │         │
│ ig_NG  │ Igbo                        │ Nigeria                           │         │
│ ii     │ Sichuan Yi                  │                                   │         │
│ ii_CN  │ Sichuan Yi                  │ China                             │         │
│ is     │ Icelandic                   │                                   │         │
│ is_IS  │ Icelandic                   │ Iceland                           │         │
│ it     │ Italian                     │                                   │         │
│ it_CH  │ Italian                     │ Switzerland                       │         │
│ it_IT  │ Italian                     │ Italy                             │         │
│ it_SM  │ Italian                     │ San Marino                        │         │
│ ja     │ Japanese                    │                                   │         │
│ ja_JP  │ Japanese                    │ Japan                             │         │
│ jgo    │ Ngomba                      │                                   │         │
│ jgo_CM │ Ngomba                      │ Cameroon                          │         │
│ jmc    │ Machame                     │                                   │         │
│ jmc_TZ │ Machame                     │ Tanzania                          │         │
│ ka     │ Georgian                    │                                   │         │
│ ka_GE  │ Georgian                    │ Georgia                           │         │
│ kab    │ Kabyle                      │                                   │         │
│ kab_DZ │ Kabyle                      │ Algeria                           │         │
│ kam    │ Kamba                       │                                   │         │
│ kam_KE │ Kamba                       │ Kenya                             │         │
│ kde    │ Makonde                     │                                   │         │
│ kde_TZ │ Makonde                     │ Tanzania                          │         │
│ kea    │ Kabuverdianu                │                                   │         │
│ kea_CV │ Kabuverdianu                │ Cape Verde                        │         │
│ khq    │ Koyra Chiini                │                                   │         │
│ khq_ML │ Koyra Chiini                │ Mali                              │         │
│ ki     │ Kikuyu                      │                                   │         │
│ ki_KE  │ Kikuyu                      │ Kenya                             │         │
│ kk     │ Kazakh                      │                                   │         │
│ kk_KZ  │ Kazakh                      │ Kazakhstan                        │         │
│ kkj    │ Kako                        │                                   │         │
│ kkj_CM │ Kako                        │ Cameroon                          │         │
│ kl     │ Kalaallisut                 │                                   │         │
│ kl_GL  │ Kalaallisut                 │ Greenland                         │         │
│ kln    │ Kalenjin                    │                                   │         │
│ kln_KE │ Kalenjin                    │ Kenya                             │         │
│ km     │ Khmer                       │                                   │         │
│ km_KH  │ Khmer                       │ Cambodia                          │         │
│ kn     │ Kannada                     │                                   │         │
│ kn_IN  │ Kannada                     │ India                             │         │
│ ko     │ Korean                      │                                   │         │
│ ko_KP  │ Korean                      │ North Korea                       │         │
│ ko_KR  │ Korean                      │ South Korea                       │         │
│ kok    │ Konkani                     │                                   │         │
│ kok_IN │ Konkani                     │ India                             │         │
│ ks     │ Kashmiri                    │                                   │         │
│ ks_IN  │ Kashmiri                    │ India                             │         │
│ ksb    │ Shambala                    │                                   │         │
│ ksb_TZ │ Shambala                    │ Tanzania                          │         │
│ ksf    │ Bafia                       │                                   │         │
│ ksf_CM │ Bafia                       │ Cameroon                          │         │
│ kw     │ Cornish                     │                                   │         │
│ kw_GB  │ Cornish                     │ United Kingdom                    │         │
│ ky     │ Kyrgyz                      │                                   │         │
│ ky_KG  │ Kyrgyz                      │ Kyrgyzstan                        │         │
│ lag    │ Langi                       │                                   │         │
│ lag_TZ │ Langi                       │ Tanzania                          │         │
│ lg     │ Ganda                       │                                   │         │
│ lg_UG  │ Ganda                       │ Uganda                            │         │
│ lkt    │ Lakota                      │                                   │         │
│ lkt_US │ Lakota                      │ United States                     │         │
│ ln     │ Lingala                     │                                   │         │
│ ln_AO  │ Lingala                     │ Angola                            │         │
│ ln_CD  │ Lingala                     │ Congo (DRC)                       │         │
│ ln_CF  │ Lingala                     │ Central African Republic          │         │
│ ln_CG  │ Lingala                     │ Congo (Republic)                  │         │
│ lo     │ Lao                         │                                   │         │
│ lo_LA  │ Lao                         │ Laos                              │         │
│ lt     │ Lithuanian                  │                                   │         │
│ lt_LT  │ Lithuanian                  │ Lithuania                         │         │
│ lu     │ Luba-Katanga                │                                   │         │
│ lu_CD  │ Luba-Katanga                │ Congo (DRC)                       │         │
│ luo    │ Luo                         │                                   │         │
│ luo_KE │ Luo                         │ Kenya                             │         │
│ luy    │ Luyia                       │                                   │         │
│ luy_KE │ Luyia                       │ Kenya                             │         │
│ lv     │ Latvian                     │                                   │         │
│ lv_LV  │ Latvian                     │ Latvia                            │         │
│ mas    │ Masai                       │                                   │         │
│ mas_KE │ Masai                       │ Kenya                             │         │
│ mas_TZ │ Masai                       │ Tanzania                          │         │
│ mer    │ Meru                        │                                   │         │
│ mer_KE │ Meru                        │ Kenya                             │         │
│ mfe    │ Morisyen                    │                                   │         │
│ mfe_MU │ Morisyen                    │ Mauritius                         │         │
│ mg     │ Malagasy                    │                                   │         │
│ mg_MG  │ Malagasy                    │ Madagascar                        │         │
│ mgh    │ Makhuwa-Meetto              │                                   │         │
│ mgh_MZ │ Makhuwa-Meetto              │ Mozambique                        │         │
│ mgo    │ Meta'                       │                                   │         │
│ mgo_CM │ Meta'                       │ Cameroon                          │         │
│ mk     │ Macedonian                  │                                   │         │
│ mk_MK  │ Macedonian                  │ Macedonia (FYROM)                 │         │
│ ml     │ Malayalam                   │                                   │         │
│ ml_IN  │ Malayalam                   │ India                             │         │
│ mn     │ Mongolian                   │                                   │         │
│ mn_MN  │ Mongolian                   │ Mongolia                          │         │
│ mr     │ Marathi                     │                                   │         │
│ mr_IN  │ Marathi                     │ India                             │         │
│ ms     │ Malay                       │                                   │         │
│ ms_BN  │ Malay                       │ Brunei                            │         │
│ ms_MY  │ Malay                       │ Malaysia                          │         │
│ ms_SG  │ Malay                       │ Singapore                         │         │
│ mt     │ Maltese                     │                                   │         │
│ mt_MT  │ Maltese                     │ Malta                             │         │
│ mua    │ Mundang                     │                                   │         │
│ mua_CM │ Mundang                     │ Cameroon                          │         │
│ my     │ Burmese                     │                                   │         │
│ my_MM  │ Burmese                     │ Myanmar (Burma)                   │         │
│ naq    │ Nama                        │                                   │         │
│ naq_NA │ Nama                        │ Namibia                           │         │
│ nb     │ Norwegian Bokmål            │                                   │         │
│ nb_NO  │ Norwegian Bokmål            │ Norway                            │         │
│ nb_SJ  │ Norwegian Bokmål            │ Svalbard and Jan Mayen            │         │
│ nd     │ North Ndebele               │                                   │         │
│ nd_ZW  │ North Ndebele               │ Zimbabwe                          │         │
│ ne     │ Nepali                      │                                   │         │
│ ne_IN  │ Nepali                      │ India                             │         │
│ ne_NP  │ Nepali                      │ Nepal                             │         │
│ nl     │ Dutch                       │                                   │         │
│ nl_AW  │ Dutch                       │ Aruba                             │         │
│ nl_BE  │ Dutch                       │ Belgium                           │         │
│ nl_BQ  │ Dutch                       │ Caribbean Netherlands             │         │
│ nl_CW  │ Dutch                       │ Curaçao                           │         │
│ nl_NL  │ Dutch                       │ Netherlands                       │         │
│ nl_SR  │ Dutch                       │ Suriname                          │         │
│ nl_SX  │ Dutch                       │ Sint Maarten                      │         │
│ nmg    │ Kwasio                      │                                   │         │
│ nmg_CM │ Kwasio                      │ Cameroon                          │         │
│ nn     │ Norwegian Nynorsk           │                                   │         │
│ nn_NO  │ Norwegian Nynorsk           │ Norway                            │         │
│ nnh    │ Ngiemboon                   │                                   │         │
│ nnh_CM │ Ngiemboon                   │ Cameroon                          │         │
│ nus    │ Nuer                        │                                   │         │
│ nus_SD │ Nuer                        │ Sudan                             │         │
│ nyn    │ Nyankole                    │                                   │         │
│ nyn_UG │ Nyankole                    │ Uganda                            │         │
│ om     │ Oromo                       │                                   │         │
│ om_ET  │ Oromo                       │ Ethiopia                          │         │
│ om_KE  │ Oromo                       │ Kenya                             │         │
│ or     │ Oriya                       │                                   │         │
│ or_IN  │ Oriya                       │ India                             │         │
│ pa     │ Punjabi                     │                                   │         │
│ pa_PK  │ Punjabi                     │ Pakistan                          │         │
│ pa_IN  │ Punjabi                     │ India                             │         │
│ pl     │ Polish                      │                                   │         │
│ pl_PL  │ Polish                      │ Poland                            │         │
│ ps     │ Pashto                      │                                   │         │
│ ps_AF  │ Pashto                      │ Afghanistan                       │         │
│ pt     │ Portuguese                  │                                   │         │
│ pt_AO  │ Portuguese                  │ Angola                            │         │
│ pt_BR  │ Portuguese                  │ Brazil                            │         │
│ pt_CV  │ Portuguese                  │ Cape Verde                        │         │
│ pt_GW  │ Portuguese                  │ Guinea-Bissau                     │         │
│ pt_MO  │ Portuguese                  │ Macau                             │         │
│ pt_MZ  │ Portuguese                  │ Mozambique                        │         │
│ pt_PT  │ Portuguese                  │ Portugal                          │         │
│ pt_ST  │ Portuguese                  │ São Tomé and Príncipe             │         │
│ pt_TL  │ Portuguese                  │ Timor-Leste                       │         │
│ rm     │ Romansh                     │                                   │         │
│ rm_CH  │ Romansh                     │ Switzerland                       │         │
│ rn     │ Rundi                       │                                   │         │
│ rn_BI  │ Rundi                       │ Burundi                           │         │
│ ro     │ Romanian                    │                                   │         │
│ ro_MD  │ Romanian                    │ Moldova                           │         │
│ ro_RO  │ Romanian                    │ Romania                           │         │
│ rof    │ Rombo                       │                                   │         │
│ rof_TZ │ Rombo                       │ Tanzania                          │         │
│ ru     │ Russian                     │                                   │         │
│ ru_BY  │ Russian                     │ Belarus                           │         │
│ ru_KG  │ Russian                     │ Kyrgyzstan                        │         │
│ ru_KZ  │ Russian                     │ Kazakhstan                        │         │
│ ru_MD  │ Russian                     │ Moldova                           │         │
│ ru_RU  │ Russian                     │ Russia                            │         │
│ ru_UA  │ Russian                     │ Ukraine                           │         │
│ rw     │ Kinyarwanda                 │                                   │         │
│ rw_RW  │ Kinyarwanda                 │ Rwanda                            │         │
│ rwk    │ Rwa                         │                                   │         │
│ rwk_TZ │ Rwa                         │ Tanzania                          │         │
│ saq    │ Samburu                     │                                   │         │
│ saq_KE │ Samburu                     │ Kenya                             │         │
│ sbp    │ Sangu                       │                                   │         │
│ sbp_TZ │ Sangu                       │ Tanzania                          │         │
│ seh    │ Sena                        │                                   │         │
│ seh_MZ │ Sena                        │ Mozambique                        │         │
│ ses    │ Koyraboro Senni             │                                   │         │
│ ses_ML │ Koyraboro Senni             │ Mali                              │         │
│ sg     │ Sango                       │                                   │         │
│ sg_CF  │ Sango                       │ Central African Republic          │         │
│ shi    │ Tachelhit                   │                                   │         │
│ shi_MA │ Tachelhit                   │ Morocco                           │         │
│ si     │ Sinhala                     │                                   │         │
│ si_LK  │ Sinhala                     │ Sri Lanka                         │         │
│ sk     │ Slovak                      │                                   │         │
│ sk_SK  │ Slovak                      │ Slovakia                          │         │
│ sl     │ Slovenian                   │                                   │         │
│ sl_SI  │ Slovenian                   │ Slovenia                          │         │
│ sn     │ Shona                       │                                   │         │
│ sn_ZW  │ Shona                       │ Zimbabwe                          │         │
│ so     │ Somali                      │                                   │         │
│ so_DJ  │ Somali                      │ Djibouti                          │         │
│ so_ET  │ Somali                      │ Ethiopia                          │         │
│ so_KE  │ Somali                      │ Kenya                             │         │
│ so_SO  │ Somali                      │ Somalia                           │         │
│ sq     │ Albanian                    │                                   │         │
│ sq_AL  │ Albanian                    │ Albania                           │         │
│ sq_MK  │ Albanian                    │ Macedonia (FYROM)                 │         │
│ sq_XK  │ Albanian                    │ Kosovo                            │         │
│ sr     │ Serbian                     │                                   │         │
│ sr_BA  │ Serbian                     │ Bosnia and Herzegovina            │         │
│ sr_ME  │ Serbian                     │ Montenegro                        │         │
│ sr_RS  │ Serbian                     │ Serbia                            │         │
│ sr_XK  │ Serbian                     │ Kosovo                            │         │
│ sv     │ Swedish                     │                                   │         │
│ sv_AX  │ Swedish                     │ Åland Islands                     │         │
│ sv_FI  │ Swedish                     │ Finland                           │         │
│ sv_SE  │ Swedish                     │ Sweden                            │         │
│ sw     │ Swahili                     │                                   │         │
│ sw_KE  │ Swahili                     │ Kenya                             │         │
│ sw_TZ  │ Swahili                     │ Tanzania                          │         │
│ sw_UG  │ Swahili                     │ Uganda                            │         │
│ swc    │ Congo Swahili               │                                   │         │
│ swc_CD │ Congo Swahili               │ Congo (DRC)                       │         │
│ ta     │ Tamil                       │                                   │         │
│ ta_IN  │ Tamil                       │ India                             │         │
│ ta_LK  │ Tamil                       │ Sri Lanka                         │         │
│ ta_MY  │ Tamil                       │ Malaysia                          │         │
│ ta_SG  │ Tamil                       │ Singapore                         │         │
│ te     │ Telugu                      │                                   │         │
│ te_IN  │ Telugu                      │ India                             │         │
│ teo    │ Teso                        │                                   │         │
│ teo_KE │ Teso                        │ Kenya                             │         │
│ teo_UG │ Teso                        │ Uganda                            │         │
│ th     │ Thai                        │                                   │         │
│ th_TH  │ Thai                        │ Thailand                          │         │
│ ti     │ Tigrinya                    │                                   │         │
│ ti_ER  │ Tigrinya                    │ Eritrea                           │         │
│ ti_ET  │ Tigrinya                    │ Ethiopia                          │         │
│ to     │ Tongan                      │                                   │         │
│ to_TO  │ Tongan                      │ Tonga                             │         │
│ tr     │ Turkish                     │                                   │         │
│ tr_CY  │ Turkish                     │ Cyprus                            │         │
│ tr_TR  │ Turkish                     │ Turkey                            │         │
│ twq    │ Tasawaq                     │                                   │         │
│ twq_NE │ Tasawaq                     │ Niger                             │         │
│ tzm    │ Central Atlas Tamazight     │                                   │         │
│ tzm_MA │ Central Atlas Tamazight     │ Morocco                           │         │
│ ug     │ Uyghur                      │                                   │         │
│ ug_CN  │ Uyghur                      │ China                             │         │
│ uk     │ Ukrainian                   │                                   │         │
│ uk_UA  │ Ukrainian                   │ Ukraine                           │         │
│ ur     │ Urdu                        │                                   │         │
│ ur_IN  │ Urdu                        │ India                             │         │
│ ur_PK  │ Urdu                        │ Pakistan                          │         │
│ uz     │ Uzbek                       │                                   │         │
│ uz_AF  │ Uzbek                       │ Afghanistan                       │         │
│ uz_UZ  │ Uzbek                       │ Uzbekistan                        │         │
│ vai    │ Vai                         │                                   │         │
│ vai_LR │ Vai                         │ Liberia                           │         │
│ vi     │ Vietnamese                  │                                   │         │
│ vi_VN  │ Vietnamese                  │ Vietnam                           │         │
│ vun    │ Vunjo                       │                                   │         │
│ vun_TZ │ Vunjo                       │ Tanzania                          │         │
│ xog    │ Soga                        │                                   │         │
│ xog_UG │ Soga                        │ Uganda                            │         │
│ yav    │ Yangben                     │                                   │         │
│ yav_CM │ Yangben                     │ Cameroon                          │         │
│ yo     │ Yoruba                      │                                   │         │
│ yo_BJ  │ Yoruba                      │ Benin                             │         │
│ yo_NG  │ Yoruba                      │ Nigeria                           │         │
│ zgh    │ Standard Moroccan Tamazight │                                   │         │
│ zgh_MA │ Standard Moroccan Tamazight │ Morocco                           │         │
│ zh     │ Chinese                     │                                   │         │
│ zh_CN  │ Chinese                     │ China                             │         │
│ zh_HK  │ Chinese                     │ Hong Kong                         │         │
│ zh_MO  │ Chinese                     │ Macau                             │         │
│ zh_SG  │ Chinese                     │ Singapore                         │         │
│ zh_TW  │ Chinese                     │ Taiwan                            │         │
│ zu     │ Zulu                        │                                   │         │
│ zu_ZA  │ Zulu                        │ South Africa                      │         │
└────────┴─────────────────────────────┴───────────────────────────────────┴─────────┘
```